docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +668,123 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handling application control actions
+
+You can define and handle each application control actions.
+
+- Defining the application control action
+
+  From Tizen 5.5, the `id` attribute is added on the `<app-control>` element. You can define the name of the application control action using the `id` attribute of the `<app-control>` element.
```

Comment:
Please make this line as 'Note' as it seems like a critical information.

Reply:
Done